src/libawkward/array/UnmaskedArray.cpp

```diff
@@ -172,7 +172,7 @@ namespace awkward {
   const Index8
   UnmaskedArray::bytemask() const {
     Index8 out(length());
-    struct Error err = awkward_zero_mask8(
+    struct Error err = kernel::zero_mask<int8_t>(
```

Comment:
Like the `_64` suffixes, this has only one variant, `awkward_zero_mask8`, so it should not be renamed/templatized.